scripts/tools/metric.yml

```diff
@@ -289,10 +289,14 @@ base: $t_base_desc_t
 members:
     - type: uint32_t
       name: "notifyEveryNReports"
-      desc: "[in,out] number of collected reports after which notification event will be signalled"
+      desc: >
+            [in,out] number of collected reports after which notification event will be signaled.
+            The driver shall update this field with the closest supported value during $tMetricStreamerOpen.
```

Comment:
@matcabral : what do we mean by "closest"? also, MetricStreamerOpen returns a handle, so to which value are we referring to?

Reply:
@jandres742  please see https://github.com/oneapi-src/level-zero-spec/issues/135 

"closest" == most approximate to what the user requested but cannot exactly be respected for the current hardware so the implementation has the flexibility to chose something else respecting user suggestion.

Reply:
Modified the wording per @jandres742 concern. @matcabral @jandres742 please review.